src/component/playground/Playground/Playground.tsx

```diff
@@ -0,0 +1,83 @@
+import { FormEventHandler, useMemo, useState, VFC } from 'react';
+import {
+    Box,
+    Button,
+    Divider,
+    Paper,
+    Typography,
+    useTheme,
+} from '@mui/material';
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { PageHeader } from 'component/common/PageHeader/PageHeader';
+import { PlaygroundConnectionFieldset } from './PlaygroundConnectionFieldset/PlaygroundConnectionFieldset';
+import { PlaygroundCodeFieldset } from './PlaygroundCodeFieldset/PlaygroundCodeFieldset';
+
+interface IPlaygroundProps {}
+
+export const Playground: VFC<IPlaygroundProps> = () => {
+    const theme = useTheme();
+    const [context, setContext] = useState<string>();
+    const [contextObject, setContextObject] = useState<string>();
+    const onSubmit: FormEventHandler<HTMLFormElement> = event => {
+        event.preventDefault();
+
+        try {
+            setContextObject(
+                JSON.stringify(JSON.parse(context || '{}'), null, 2)
+            );
+        } catch (e) {}
+    };
+
+    return (
+        <PageContent header={<PageHeader title="Unleash playground" />}>
+            <Paper
+                elevation={0}
+                sx={{
+                    px: 4,
+                    py: 3,
+                    background: theme.palette.grey[200],
+                }}
+            >
+                <Box component="form" onSubmit={onSubmit}>
+                    <Typography
+                        sx={{
+                            mb: 3,
+                        }}
+                    >
+                        Configure playground
+                    </Typography>
+                    <PlaygroundConnectionFieldset />
+                    <Divider
```

Comment:
Could this be a `StyledDashedDivider`, then we could reuse it later and we wouldn't have to repeat the styles beneath. Might clean up the render function a bit.

Reply:
I'll add a variant to theme.

Reply:
First divider is dashed, the second one isn't